mu/app.py

```diff
@@ -270,9 +302,16 @@ def run():
     setup_logging()
     logging.info("\n\n-----------------\n\nStarting Mu {}".format(__version__))
     logging.info(platform.uname())
+    logging.info("Process id: {}".format(os.getpid()))
     logging.info("Platform: {}".format(platform.platform()))
     logging.info("Python path: {}".format(sys.path))
     logging.info("Language code: {}".format(i18n.language_code))
+    try:
+        _shared_memory.acquire()
+    except MutexError as exc:
+        [message] = exc.args
+        logging.error(message)
+        sys.exit()
```

Comment:
Based on the description in https://github.com/mu-editor/mu/pull/2297#issuecomment-1186488744, should this be: 
```suggestion
        sys.exit(2)
```